plugins/nodes/src/js/components/HealthTab.js

```diff
@@ -22,6 +23,7 @@ class HealthTab extends React.Component {
   constructor() {
     super(...arguments);
 
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate;
```

Comment:
Are you sure we don't have to bind this? Seems like it might always return true because it might read `this.state` and `this.props` as `undefined`